docs/storybook/pages/wall.stories.tsx

```
@@ -0,0 +1,389 @@
+                    'https://www.bouyguestelecom.fr/catalogue/image-resize/full/media/catalog/product/v/i/vig-samsung-galaxy-s23-fe-graphite-face_1.png'
+                  }
+                />
+              </Columns>
+              <Pagination length={10} pageSize={5} />
+            </Column>
+          </Columns>
+        </Section>
+        <Section>
+          <Container>
+            <Accordion children>
+              <AccordionItem children open>
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)